packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-header/pipeline-actions.tsx

```diff
@@ -32,16 +32,31 @@ type PipelineActionsProps = {
   isOptionsVisible: boolean;
   onRunAggregation: () => void;
   onToggleOptions: () => void;
+  onExportAggregationResults: () => void;
 };
 
 export const PipelineActions: React.FunctionComponent<PipelineActionsProps> = ({
   isOptionsVisible,
   onRunAggregation,
   onToggleOptions,
+  onExportAggregationResults
 }) => {
   const optionsIcon = isOptionsVisible ? 'CaretDown' : 'CaretRight';
+  const showExportButton = process.env.COMPASS_ENABLE_AGGREGATION_EXPORT === 'true'
```

Comment:
Since `compass-aggregations` is embedded in cloud/browser, how do you feel about making these `process.env` checks undefined safe:
```suggestion
  const showExportButton = process?.env?.COMPASS_ENABLE_AGGREGATION_EXPORT === 'true'
```

Reply:
Thinking about this more I wonder if there's a way we could be testing the `compass-aggregations` (and maybe some other packages) on more real browser test environments to surface things like this more. Maybe worth a ticket down the line?

Reply:
It's worth having the optional chaining here, but this will be processed by webpack (twice) and by default webpack will provide `process` for the browser environment (or in some cases will remove these expressions based on the configuration, although I'm not sure if optional chaining is something they handle in this case)